Sources/BraintreeCard/Public/BraintreeCard/BTCardNonce.h

```diff
@@ -6,6 +6,7 @@
 
 @class BTThreeDSecureInfo;
 @class BTAuthenticationInsight;
+@class BTBinData;
```

Comment:
I see a handful of files have new added `@class` directives (other examples in the PR: BTConfiguration and BTPostalAddress). In this example of the `BTCardNonce.h`, it doesn't look like any changes to `BTCardNonce.m` were made. What were these new directives added for?

Reply:
I think some of these can be cleaned up. I'll go through and revert any ones that aren't needed. 👍

Reply:
Updated - the ones remaining are needed because of deleting the references to the apiclient.h files which handled some of the forward declarations and imports for other files previously